app/retail/templates/shared/footer_scripts.html

```diff
@@ -15,24 +15,18 @@
     along with this program. If not, see <http://www.gnu.org/licenses/>.
 
 {% endcomment %}
-{% load static %}
+{% load static compress %}
 <script type="text/javascript">
   var static_url = "{{ STATIC_URL }}";
   var media_url = "{{ MEDIA_URL }}";
 </script>
-<script type="text/javascript" src="{% url 'javascript-catalog' %}"></script>
+{% compress js %}
+<script src="{% static "jsi18n/en/djangojs.js" %}"></script>
 <script src="{% static "v2/js/lib/jquery.js" %}"></script>
-{% if user.is_authenticated and env == 'prod' %}
-  <script src="https://cdn.jsdelivr.net/npm/vue"></script>
-{% elif user.is_authenticated %}
-  <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
-{% endif %}
-{% include 'shared/sentry.html' %}
+<script src="{% static "v2/js/lib/vue.js" %}"></script>
```

Comment:
can we add <script src="https://cdn.jsdelivr.net/npm/vue"></script> to not prod environment ? that is used to debug, so will show the correct vue errors in localhost

Reply:
yes that makes sense

Reply:
087618be7d7c9bde204057928e94253b202f4476